### PR TITLE
Cut update

### PR DESCRIPTION
**Pull request by MaxNoe**
* Use the 40 % efficiency cut as done by event display
* Add `min_events` parameter to the cut function calculation that requires at least that number of events to calculate the cut.
* Add smoothing option
* Add support for columns without units in `calculate_percentile_cut`

### CODE REVIEW

**Comment by LukasNickel** [comment]
How are the 40% motivated? I thought eventdisplay uses 80%?

File: examples/calculate_eventdisplay_irfs.py [hunk 1]
```diff
@@ -109,6 +109,8 @@ def main():
     background = table.vstack(
         [particles["proton"]["events"], particles["electron"]["events"]]
     )
+
+    INITIAL_GH_CUT = np.quantile(gammas['gh_score'], (1 - INITIAL_GH_CUT_EFFICENCY))
```
Comment (HealthyPear) [inline examples/calculate_eventdisplay_irfs.py:113]: wasn't this supposed to be a simple fixed cut?

**Comment by HealthyPear** [comment]
> How are the 40% motivated? I thought eventdisplay uses 80%?

That is the maximum efficiency allowed later, this is instead a first global cut

File: examples/calculate_eventdisplay_irfs.py (hunk 1 above)
Comment (MaxNoe) [reply examples/calculate_eventdisplay_irfs.py:113]: Yes, it is a single cut. The global cut which has 40 % efficiency. This calculates a single cut with that efficiency.

**Review by HealthyPear — APPROVED** [review]

**Pull request merged**